PowerEditor/installer/nativeLang/estonian.xml

```diff
@@ -1,903 +1,1008 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-*Estonian language file for Notepad++*
-
-	** Created By:- Yusuf Mullaji *
-		Email Id-> yusufmullaji@rocketmail.com
-
+Estonian translation by Andres Traks
+https://github.com/AndresTraks/
```

Comment:
We need the ref of people (email) who know the language to maintain the translation. We don't need the ref of people who use the translator.

@yusuf2526 
Your contribution wouldn't be accepted If I knew it has been done by translator .
You can always do your contribution on the language you speak.
Otherwise, if you want to to satisfy your ego, please look for somewhere-else - Notepad++ is not the project for that.